src/app.config.json.tpl

```diff
@@ -1,22 +1,22 @@
 {
   "$schema": "../node_modules/@alfresco/adf-core/app.config.schema.json",
-  "ecmHost": "{protocol}//{hostname}{:port}",
-  "aosHost": "{protocol}//{hostname}{:port}/alfresco/aos",
-  "baseShareUrl": "{protocol}//{hostname}{:port}/#/preview/s",
-  "providers": "ECM",
-  "authType": "BASIC",
+  "ecmHost": "${APP_CONFIG_ECM_HOST}",
```

Comment:
Can you please specify these default values in the README.md file? From now on the developer need to have these inside their .env file.

Reply:
yep sure